internal/store/organizations.go

```diff
@@ -55,13 +56,15 @@ func (s *Store) Organizations(opts *atlas.OrganizationsListOptions) (*atlas.Orga
 }
 
 // Organization encapsulate the logic to manage different cloud providers.
-func (s *Store) Organization(id string) (*atlas.Organization, error) {
+func (s *Store) Organization(id string) (*v1alpha.ApiOrganizationView, error) {
```

Comment:
Is `*v1alpha.ApiOrganizationView` coming from the autogenerated sdk? Could we use v2 for all the autogenerated code?

Reply:
> Is *v1alpha.ApiOrganizationView coming from the autogenerated sdk?
Yes. v1alpha just import name that we can change. See other thread.

There is a lot of confusion between golang client version and Atlas API version.
Atlas V2 API client is currently in v1alpha version etc.

Reply:
oh man, this is really hard to follow 💔

Reply:
@fmenezes what improvements would you suggest here? We are the first users of the SDK and have a great chance to improve it before impacting our customers.

I'd recommend a cross team conversation here to apply what we're learning and improve the SDK for everyone.